_custombuild

```diff
@@ -53,17 +53,13 @@ var components = gulp.src([
   APP_ROOT + 'components/gaia-dialog/gaia-dialog.js',
```

Comment:
Actually, that looks like what you did here. But if I'm mistaken and you didn't, that would be a good follow-up.